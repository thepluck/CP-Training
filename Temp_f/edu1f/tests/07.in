1000 40
0 2
1 0
2 2
3 0
4 2
5 0
6 2
7 0
8 2
9 0
10 2
11 0
12 2
13 0
14 2
15 0
16 2
17 0
18 2
19 0
20 2
21 0
22 2
23 0
24 2
25 0
26 2
27 0
28 2
29 0
30 2
31 0
32 2
33 0
34 2
35 0
36 2
37 0
38 2
39 0
40 2
41 0
42 2
43 0
44 2
45 0
46 2
47 0
48 2
49 0
50 2
51 0
52 2
53 0
54 2
55 0
56 2
57 0
58 2
59 0
60 2
61 0
62 2
63 0
64 2
65 0
66 2
67 0
68 2
69 0
70 2
71 0
72 2
73 0
74 2
75 0
76 2
77 0
78 2
79 0
80 2
81 0
82 2
83 0
84 2
85 0
86 2
87 0
88 2
89 0
90 2
91 0
92 2
93 0
94 2
95 0
96 2
97 0
98 2
99 0
100 2
101 0
102 2
103 0
104 2
105 0
106 2
107 0
108 2
109 0
110 2
111 0
112 2
113 0
114 2
115 0
116 2
117 0
118 2
119 0
120 2
121 0
122 2
123 0
124 2
125 0
126 2
127 0
128 2
129 0
130 2
131 0
132 2
133 0
134 2
135 0
136 2
137 0
138 2
139 0
140 2
141 0
142 2
143 0
144 2
145 0
146 2
147 0
148 2
149 0
150 2
151 0
152 2
153 0
154 2
155 0
156 2
157 0
158 2
159 0
160 2
161 0
162 2
163 0
164 2
165 0
166 2
167 0
168 2
169 0
170 2
171 0
172 2
173 0
174 2
175 0
176 2
177 0
178 2
179 0
180 2
181 0
182 2
183 0
184 2
185 0
186 2
187 0
188 2
189 0
190 2
191 0
192 2
193 0
194 2
195 0
196 2
197 0
198 2
199 0
200 2
201 0
202 2
203 0
204 2
205 0
206 2
207 0
208 2
209 0
210 2
211 0
212 2
213 0
214 2
215 0
216 2
217 0
218 2
219 0
220 2
221 0
222 2
223 0
224 2
225 0
226 2
227 0
228 2
229 0
230 2
231 0
232 2
233 0
234 2
235 0
236 2
237 0
238 2
239 0
240 2
241 0
242 2
243 0
244 2
245 0
246 2
247 0
248 2
249 0
250 2
251 0
252 2
253 0
254 2
255 0
256 2
257 0
258 2
259 0
260 2
261 0
262 2
263 0
264 2
265 0
266 2
267 0
268 2
269 0
270 2
271 0
272 2
273 0
274 2
275 0
276 2
277 0
278 2
279 0
280 2
281 0
282 2
283 0
284 2
285 0
286 2
287 0
288 2
289 0
290 2
291 0
292 2
293 0
294 2
295 0
296 2
297 0
298 2
299 0
300 2
301 0
302 2
303 0
304 2
305 0
306 2
307 0
308 2
309 0
310 2
311 0
312 2
313 0
314 2
315 0
316 2
317 0
318 2
319 0
320 2
321 0
322 2
323 0
324 2
325 0
326 2
327 0
328 2
329 0
330 2
331 0
332 2
333 0
334 2
335 0
336 2
337 0
338 2
339 0
340 2
341 0
342 2
343 0
344 2
345 0
346 2
347 0
348 2
349 0
350 2
351 0
352 2
353 0
354 2
355 0
356 2
357 0
358 2
359 0
360 2
361 0
362 2
363 0
364 2
365 0
366 2
367 0
368 2
369 0
370 2
371 0
372 2
373 0
374 2
375 0
376 2
377 0
378 2
379 0
380 2
381 0
382 2
383 0
384 2
385 0
386 2
387 0
388 2
389 0
390 2
391 0
392 2
393 0
394 2
395 0
396 2
397 0
398 2
399 0
400 2
401 0
402 2
403 0
404 2
405 0
406 2
407 0
408 2
409 0
410 2
411 0
412 2
413 0
414 2
415 0
416 2
417 0
418 2
419 0
420 2
421 0
422 2
423 0
424 2
425 0
426 2
427 0
428 2
429 0
430 2
431 0
432 2
433 0
434 2
435 0
436 2
437 0
438 2
439 0
440 2
441 0
442 2
443 0
444 2
445 0
446 2
447 0
448 2
449 0
450 2
451 0
452 2
453 0
454 2
455 0
456 2
457 0
458 2
459 0
460 2
461 0
462 2
463 0
464 2
465 0
466 2
467 0
468 2
469 0
470 2
471 0
472 2
473 0
474 2
475 0
476 2
477 0
478 2
479 0
480 2
481 0
482 2
483 0
484 2
485 0
486 2
487 0
488 2
489 0
490 2
491 0
492 2
493 0
494 2
495 0
496 2
497 0
498 2
499 0
500 2
501 0
502 2
503 0
504 2
505 0
506 2
507 0
508 2
509 0
510 2
511 0
512 2
513 0
514 2
515 0
516 2
517 0
518 2
519 0
520 2
521 0
522 2
523 0
524 2
525 0
526 2
527 0
528 2
529 0
530 2
531 0
532 2
533 0
534 2
535 0
536 2
537 0
538 2
539 0
540 2
541 0
542 2
543 0
544 2
545 0
546 2
547 0
548 2
549 0
550 2
551 0
552 2
553 0
554 2
555 0
556 2
557 0
558 2
559 0
560 2
561 0
562 2
563 0
564 2
565 0
566 2
567 0
568 2
569 0
570 2
571 0
572 2
573 0
574 2
575 0
576 2
577 0
578 2
579 0
580 2
581 0
582 2
583 0
584 2
585 0
586 2
587 0
588 2
589 0
590 2
591 0
592 2
593 0
594 2
595 0
596 2
597 0
598 2
599 0
600 2
601 0
602 2
603 0
604 2
605 0
606 2
607 0
608 2
609 0
610 2
611 0
612 2
613 0
614 2
615 0
616 2
617 0
618 2
619 0
620 2
621 0
622 2
623 0
624 2
625 0
626 2
627 0
628 2
629 0
630 2
631 0
632 2
633 0
634 2
635 0
636 2
637 0
638 2
639 0
640 2
641 0
642 2
643 0
644 2
645 0
646 2
647 0
648 2
649 0
650 2
651 0
652 2
653 0
654 2
655 0
656 2
657 0
658 2
659 0
660 2
661 0
662 2
663 0
664 2
665 0
666 2
667 0
668 2
669 0
670 2
671 0
672 2
673 0
674 2
675 0
676 2
677 0
678 2
679 0
680 2
681 0
682 2
683 0
684 2
685 0
686 2
687 0
688 2
689 0
690 2
691 0
692 2
693 0
694 2
695 0
696 2
697 0
698 2
699 0
700 2
701 0
702 2
703 0
704 2
705 0
706 2
707 0
708 2
709 0
710 2
711 0
712 2
713 0
714 2
715 0
716 2
717 0
718 2
719 0
720 2
721 0
722 2
723 0
724 2
725 0
726 2
727 0
728 2
729 0
730 2
731 0
732 2
733 0
734 2
735 0
736 2
737 0
738 2
739 0
740 2
741 0
742 2
743 0
744 2
745 0
746 2
747 0
748 2
749 0
750 2
751 0
752 2
753 0
754 2
755 0
756 2
757 0
758 2
759 0
760 2
761 0
762 2
763 0
764 2
765 0
766 2
767 0
768 2
769 0
770 2
771 0
772 2
773 0
774 2
775 0
776 2
777 0
778 2
779 0
780 2
781 0
782 2
783 0
784 2
785 0
786 2
787 0
788 2
789 0
790 2
791 0
792 2
793 0
794 2
795 0
796 2
797 0
798 2
799 0
800 2
801 0
802 2
803 0
804 2
805 0
806 2
807 0
808 2
809 0
810 2
811 0
812 2
813 0
814 2
815 0
816 2
817 0
818 2
819 0
820 2
821 0
822 2
823 0
824 2
825 0
826 2
827 0
828 2
829 0
830 2
831 0
832 2
833 0
834 2
835 0
836 2
837 0
838 2
839 0
840 2
841 0
842 2
843 0
844 2
845 0
846 2
847 0
848 2
849 0
850 2
851 0
852 2
853 0
854 2
855 0
856 2
857 0
858 2
859 0
860 2
861 0
862 2
863 0
864 2
865 0
866 2
867 0
868 2
869 0
870 2
871 0
872 2
873 0
874 2
875 0
876 2
877 0
878 2
879 0
880 2
881 0
882 2
883 0
884 2
885 0
886 2
887 0
888 2
889 0
890 2
891 0
892 2
893 0
894 2
895 0
896 2
897 0
898 2
899 0
900 2
901 0
902 2
903 0
904 2
905 0
906 2
907 0
908 2
909 0
910 2
911 0
912 2
913 0
914 2
915 0
916 2
917 0
918 2
919 0
920 2
921 0
922 2
923 0
924 2
925 0
926 2
927 0
928 2
929 0
930 2
931 0
932 2
933 0
934 2
935 0
936 2
937 0
938 2
939 0
940 2
941 0
942 2
943 0
944 2
945 0
946 2
947 0
948 2
949 0
950 2
951 0
952 2
953 0
954 2
955 0
956 2
957 0
958 2
959 0
960 2
961 0
962 2
963 0
964 2
965 0
966 2
967 0
968 2
969 0
970 2
971 0
972 2
973 0
974 2
975 0
976 2
977 0
978 2
979 0
980 2
981 0
982 2
983 0
984 2
985 0
986 2
987 0
988 2
989 0
990 2
991 0
992 2
993 0
994 2
995 0
996 2
997 0
1000 -1
0 -1
-1 1 1001 1
-2 1 1002 1
-3 1 1003 1
-4 1 1004 1
-5 1 1005 1
-6 1 1006 1
-7 1 1007 1
-8 1 1008 1
-9 1 1009 1
-10 1 1010 1
-11 1 1011 1
-12 1 1012 1
-13 1 1013 1
-14 1 1014 1
-15 1 1015 1
-16 1 1016 1
-17 1 1017 1
-18 1 1018 1
-19 1 1019 1
-20 1 1020 1
-21 1 1021 1
-22 1 1022 1
-23 1 1023 1
-24 1 1024 1
-25 1 1025 1
-26 1 1026 1
-27 1 1027 1
-28 1 1028 1
-29 1 1029 1
-30 1 1030 1
-31 1 1031 1
-32 1 1032 1
-33 1 1033 1
-34 1 1034 1
-35 1 1035 1
-36 1 1036 1
-37 1 1037 1
-38 1 1038 1
-39 1 1039 1
-40 1 1040 1
