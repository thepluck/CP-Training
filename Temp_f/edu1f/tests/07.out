498.500000000000
498.500000000000
498.500000000000
498.500000000000
498.500000000000
498.500000000000
498.500000000000
498.500000000000
498.500000000000
498.500000000000
498.500000000000
498.500000000000
498.500000000000
498.500000000000
498.500000000000
498.500000000000
498.500000000000
498.500000000000
498.500000000000
498.500000000000
498.500000000000
498.500000000000
498.500000000000
498.500000000000
498.500000000000
498.500000000000
498.500000000000
498.500000000000
498.500000000000
498.500000000000
498.500000000000
498.500000000000
498.500000000000
498.500000000000
498.500000000000
498.500000000000
498.500000000000
498.500000000000
498.500000000000
498.500000000000
